### PR TITLE
Added endpoint to repsvc client

### PR DESCRIPTION
**Pull request by codynhat**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@Library('hiya-jenkins-helpers')
+def context = new com.hiya.JenkinsContextFactory().getContext()
+import com.hiya.JenkinsfileTemplates
+JenkinsfileTemplates.basicSbtFoundationProject(context)
```
Comment (ayberk) [inline Jenkinsfile]: Nice :)

File: clients/src/main/scala/com/hiya/gateway/clients/reputation/RepSvcClient.scala [hunk 2]
```diff
@@ -77,19 +77,19 @@ case class RepSvcClient(host: String, port: Int, configTimeout: FiniteDuration =
   def getTopSpammers(req: GetTopSpammers.Request)(implicit system: ActorSystem, ec: ExecutionContext, id: SequencedTrackingId): Future[GetTopSpammers.Response] = {
     val url = formatUrl("get_top_spammers")
     val pipeline = callPipeline[GetTopSpammers.Response]("repsvc","get_top_spammers","POST")
-    pipeline(Post(url, req.toJson))
```
Comment (ayberk) [inline clients/src/main/scala/com/hiya/gateway/clients/reputation/RepSvcClient.scala]: What exactly changed here? Are we importing implicit conversions, or pipeline doesn't expect json anymore?

Comment (codynhat) [inline clients/src/main/scala/com/hiya/gateway/clients/reputation/RepSvcClient.scala]: The pipeline needs something with an implicit SprayMarshaller. I thought it was redundant to do .toJson

**Comment by ayberk** [comment]
LGTM

**Pull request merged**